lib/services/schedule.js

```diff
@@ -13,26 +14,34 @@ class GoogleCal {
 
   findNextStreamDay() {
     return this.getListOfUpcomingEvents(this.configuration.GOOGLE_CALENDAR_ID).then((events) => {
-      const now = moment.tz('UTC');
       const nextEvent = events.find((event) => {
-        if (event.start.date !== undefined) {
+        if (event.start !== undefined) {
           return true;
         }
-
-        return !(
-          moment.tz(event.end.dateTime, 'YYYY-MM-DDTHH:mm:ss', event.end.timeZone).isAfter(now) &&
-          moment.tz(event.start.dateTime, 'YYYY-MM-DDTHH:mm:ss', event.start.timeZone).isBefore(now)
-        );
       });
-
       if (!nextEvent) {
         return null;
       }
-
+      
+      const startTime = moment.tz(nextEvent.start.dateTime, 'YYYY-MM-DDTHH:mm:ss', this.timezone)
```

Comment:
The issue does say to use the broadcaster's timezone as configured in the bot, but I no longer think that's the right call. The broadcaster's timezone and the timezone of their calendar might differ. If that's the case, these dates won't be parsed correctly. For instance, Destiny lives in `America/New_York`, but his calendar is in `America/Chicago`.

Let's revert to using the calendar's timezone. It's a little annoying that we only get a timezone for non-all-day events, however. I was originally going to suggest you use [this API](https://developers.google.com/calendar/api/v3/reference/calendars/get), but that requires authorization. Considering the calendar timezone will change rarely (if at all), maybe an easier approach is to introduce a new config named `calendarTimezone`.

Reply:
I just realized that [this API](https://developers.google.com/calendar/api/v3/reference/events/list) (the one we use for events) accepts a timezone as a parameter. Using this parameter probably the best solution. You can just get all the dates/times in UTC and that would remove timezones from the equation altogether.

Reply:
I've added the timezone parameter, then used .utc in the schedule service and then just regular moments in the command implementation I believe this is the intent right? so we grab it as a utc date from the calendar then convert to the local timezone.